orchestrate/obase/pipelines.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 type outputWorkerSettings struct {
-	bufferer   base.ChunkBufferer
-	serializer base.LogSerializer
-	chunkMaker base.LogChunkMaker
-	consumer   base.ChunkConsumer
+	Bufferer   base.ChunkBufferer
+	Serializer base.LogSerializer
+	ChunkMaker base.LogChunkMaker
+	Consumer   base.ChunkConsumer
```

Comment:
Why export the fields though? This structure is internal to this package, isn't it?

Reply:
fixed